ChangeLog.d/pkcs7-parser.txt

```diff
@@ -0,0 +1,15 @@
+Features
+   * Added partial support for parsing the PKCS #7 cryptographic message
+     syntax, as defined in RFC 2315. Currently, support is limited to the
+     following:
+     - Only the signed data content type, version 1 is supported.
+     - Only DER encoding is supported.
+     - Only a single digest algorithm per message is supported.
+     - Only 0 or 1 certificates are supported per message, which must be in
```

Comment:
This sounds clunky (plural certificates is correct for 0 but not for 1). How about

```
     - Certificates must be in X.509 format. A message must have either one
       certificate or no certificates.
```